multivariable-calculus/manipulating-derivatives.md

```diff
@@ -171,3 +171,55 @@ Putting it all together:
 This total differential is now suitable for integration with respect to *T*
 and *P* using only measurable quantities!
 ```
+
+
+# Skill Builder:
+
+### Given: 
+\begin{equation}
+\d{f} = p \d{x} - q \d{y}
+\end{equation}
+
+$
+f=cos(4x+y^2)+x^2y
+$
+
+
+1. 
+$
+\td{}{x}{f}{y} =
+$
+
+```{solution}
+$$
+\td{}{x}{f}{y} = \frac{1}{\td{}{f}{x}{y}} = 1/p = \left(-4sin(4x+y^2)+ 2xy\right)^{-1}
+$$
```

Comment:
```suggestion
\begin{align}
\td{}{x}{f}{y} &= \frac{1}{\td{}{f}{x}{y}} \\
 1/p &= \left(-4\sin(4x+y^2)+ 2xy\right)^{-1}
\end{align}
```

These expressions should be written on new lines so they don't take up so much horizontal space. Make similar edits to the rest of the solutions. 

If an equation takes up an entire line, it should be wrapped in `\begin{equation}\end{equation}` and not in $$. If an equation takes up multiple lines, it should be wrapped in `\being{align}\end{align}`.

Also replace all instances of `sin` and `cos` with `\sin` and `\cos` so those functions are formatted properly.